indexer/config/config.go

```diff
@@ -68,8 +76,11 @@ func LoadConfig(path string) (Config, error) {
 
 	// Decode the TOML data.
 	if _, err := toml.Decode(string(data), &conf); err != nil {
+		log.Info("Failed to decode config file", "message", err)
```

Comment:
Lets place this at the error level and rename "message" to "err" to be consistent with how we log errors elsehwere

Reply:
```suggestion
		log.Error("Failed to decode config file", "err", err)
```